src/app/public/matchers/matchers.ts

```diff
@@ -124,7 +124,7 @@ const matchers: jasmine.CustomMatcherFactories = {
             }
 
             message.push(
-              `Expected element not to have CSS style "${styleName}: ${expectedStyle}"`
+              `Expected element did not have CSS style "${styleName}: ${expectedStyle}"`
```

Comment:
This changes the meaning of the message. Its intent is to inform the caller that the specified element was expected to not have the specified CSS style, but it did have the specified style. The original message should be left intact.

Reply:
OK, I guess I misinterpreted this (and thought it was just poor wording) as I was trying to understand the code for writing the interface descriptions.  I changed it back.